politeiawww/config.go

```diff
@@ -43,11 +44,10 @@ var (
 	defaultRPCCertFile   = filepath.Join(sharedconfig.DefaultHomeDir, "rpc.cert")
 	defaultCookieKeyFile = filepath.Join(sharedconfig.DefaultHomeDir, "cookie.key")
 	defaultLogDir        = filepath.Join(sharedconfig.DefaultHomeDir, defaultLogDirname)
+	defaultTemplateDir   = filepath.Join(sharedconfig.DefaultHomeDir, defaultTemplateDirname)
 
-	templateNewUserEmail = template.Must(
-		template.New("new_user_email_template").Parse(templateNewUserEmailRaw))
-	templateResetPasswordEmail = template.Must(
-		template.New("reset_password_email_template").Parse(templateResetPasswordEmailRaw))
+	templateNewUserEmail       = template.Must(template.ParseFiles(filepath.Join(defaultTemplateDir, "newuser.html")))
```

Comment:
The templates are under `politeiawww/templates/email/` but this code expects them to be copied into `<config dir>/politeiawww/templates/`. I think it would be better to stay consistent with the `templates/email/` structure.

Reply:
I agree, it's much easier and I've seen a lot of repos following that convention